core/trino-parser/src/main/java/io/trino/sql/tree/QualifiedName.java

```diff
@@ -48,7 +48,7 @@ public static QualifiedName of(String name)
     public static QualifiedName of(Iterable<Identifier> originalParts)
     {
         requireNonNull(originalParts, "originalParts is null");
-        checkArgument(!isEmpty(originalParts), "originalParts is empty");
+        checkArgument(stream(originalParts).findAny().isPresent(), "originalParts is empty");
```

Comment:
`isEmpty` kinda looked nicer :/ It conveys the intention in a more straightforward way.